src/main/java/edu/cornell/kfs/sys/document/web/struts/CuFinancialMaintenanceDocumentAction.java

```diff
@@ -102,8 +102,7 @@ public ActionForward insertBONote(ActionMapping mapping, ActionForm form, HttpSe
             }
         }
 
-        DataDictionary dataDictionary = getDataDictionaryService().getDataDictionary();
-        DocumentEntry entry = dataDictionary.getDocumentEntry(document.getClass().getName());
+        DocumentEntry entry = getDocumentDictionaryService().getDocumentEntry(document.getClass().getName());

```

Comment:
Another spot that might need updating to call a different service method.

Reply:
update